tests/unit/dom/blur-test.js

```diff
@@ -0,0 +1,68 @@
+import { module, test } from 'qunit';
+import { focus, blur, setContext, unsetContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+
+module('DOM Helper: blur', function(hooks) {
+  let element;
+
+  hooks.beforeEach(async function(assert) {
+    // used to simulate how `setupRenderingTest` (and soon `setupApplicationTest`)
+    // set context.element to the rootElement
+    this.element = document.querySelector('#qunit-fixture');
```

Comment:
I'm wondering if we should use an explicit `context` object here instead of `this` 🤔